src/veritysetup/veritysetup-generator.c

```diff
@@ -158,6 +164,16 @@ static int parse_proc_cmdline_item(const char *key, const char *value, void *dat
                 r = free_and_strdup(&arg_hash_what, value);
                 if (r < 0)
                         return log_oom();
+
+        } else if (proc_cmdline_key_streq(key, "systemd.verity.options")) {
+
+                if (proc_cmdline_value_missing(key, value))
+                        return 0;
+
+                r = free_and_strdup(&arg_options, value);
+                if (r < 0)
+                        return log_oom();
+
```

Comment:
doesn#t matter, but this empty line seems too much ;-)

Reply:
Sure, tbh, I have added that line because that same empty line exists in the other if statement above ;)

I can drop them as well in another commit.